lib/features/mailbox_dashboard/presentation/controller/search_controller.dart

```diff
@@ -157,7 +157,6 @@ class SearchController extends BaseController {
         searchInputController.clear();
         cleanSearchFilter();
         searchFocus.unfocus();
-        searchState.value = searchState.value.disableSearchState();
```

Comment:
in order to receive onTap event from search icon in search bar, searchState have not to be disabled here but later in clearSearchFilter, is it ok that i remove this line?

Reply:
If you need to remove it, you should double check the cases when searching.